howtos/how-to-add-a-create_new_chat-button-in-talkjs/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
I can see a warning in the console about using this in production - I think it's fine for a demo but it would be good to link to https://tailwindcss.com/docs/installation/play-cdn in the tutorial text at the point where you talk about styling

Reply:
Done: Updated the note section at https://docs.google.com/document/d/1d4EZ2HTq6yj8POuDjnCnMjBbd_RZ-aYNQYRR3lh1eUo/edit#heading=h.j2wn07dei4xo